components/atoms/ToggleGroup/toggle-group.tsx

```diff
@@ -0,0 +1,59 @@
+import * as GroupToggle from "@radix-ui/react-toggle-group";
+import { useState } from "react";
+import ToggleGroupItem from "../ToggleGroupItem/toggle-group-item";
+
+interface ToggleGroupProps {
+    children: React.ReactNode,
+    /** Allows the user to deselect an option, resulting in none being selected. */
+    allowNone?: boolean,
+    /** Decides which option will be selected by default. Provide the position of the element to be selected starting from 0. */
+    defaultSelection?: string | number,
+    /** Callback function that is called when the user selects an option, and the position of the selected element is passed into as string, starting with '0'. */
+    handleChange?: (value: string) => void
+}
+
+/** A ToggleGroup component that allows the user to select one option from a list of options.
+ * Warning: Do NOT re-order the children after the component has been rendered.
+ *
+ * @example
+ * <ToggleGroup>
+ *      <>Option 1</> // React Fragment. Will use our styling
+ *      <>Option <span>2</span></> // Can be a nested component
+ *      <div>Option 3</div> // Any type of component can be used
+ * </ToggleGroup>
+ */
+const ToggleGroup = ({
```

Comment:
This component needs an example added to the Storybook also. There are example stories are here https://github.com/open-sauced/insights/tree/beta/stories/atoms

Reply:
Added ToggleGroup to Storybook. However ToggleGroupItem can't be added as a standalone component for test as it will give error if not put inside a ToggleGroup component, but it is included in ToggleGroup tests anyways